### PR TITLE
Royal KNIGHTS

### PR DESCRIPTION
**Pull request by SELFHELL**
## About The Pull Request

Reworks the Royal Guards into proper KNIGHTS

Also they get a fancy title and cool armor.

peak

## Why It's Good For The Game:

Improves the frankly... poor defense provided to the king. When atleast 5-6 random ass adventurers are spawning with full plate armor and bastard swords, while the king has what amounts to a town guard with a sword, there is a MAJOR ISSUE. 

## Pre-Merge Checklist

- [x] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

### CODE REVIEW

**Comment by guayitoo** [comment]
I think we should reduce the amount of combat pilgrims rather than buff the strength of the guard until there's no point of having smiths tbh.

File: code/modules/jobs/job_types/roguetown/garrison/manorguard.dm [hunk 1]
```diff
@@ -29,44 +29,52 @@
...
 			var/index = findtext(H.real_name, " ")
 			if(index)
 				index = copytext(H.real_name, 1,index)
 			if(!index)
 				index = H.real_name
-			S.name = "guard's tabard ([index])"
+			S.name = "knight tabard ([index])"
+		var/prev_real_name = H.real_name
+		var/prev_name = H.name
+		var/honorary = "Sir"
```
Comment (BadAtThisGame302) [inline code/modules/jobs/job_types/roguetown/garrison/manorguard.dm:42]: Ser.

File: code/modules/jobs/job_types/roguetown/garrison/manorguard.dm [hunk 2]
```diff
@@ -29,44 +29,52 @@
...
-		H.change_stat("endurance", 1)
-		H.change_stat("speed", 1)
+		H.mind.adjust_skillrank(/datum/skill/combat/crossbows, 4, TRUE)
+		H.mind.adjust_skillrank(/datum/skill/combat/bows, 3, TRUE)
+		H.mind.adjust_skillrank(/datum/skill/misc/athletics, 3, TRUE)
+		H.mind.adjust_skillrank(/datum/skill/misc/climbing, 1, TRUE)
+		H.mind.adjust_skillrank(/datum/skill/misc/reading, 3, TRUE)
+		H.mind.adjust_skillrank(/datum/skill/misc/riding, 4, TRUE)
+		backl = /obj/item/rogueweapon/sword/long
+		H.change_stat("strength", 5)
```
Comment (1N7Slayer) [inline code/modules/jobs/job_types/roguetown/garrison/manorguard.dm:75]: close this joke PR instantly

**Comment by 1N7Slayer** [comment]
The stat buffs are ridiculous, as previously said. Every character in a role in the game needs to be mortal and vulnerable. Giving the RG absurd stat tweaks are not going to enhance gameplay or roleplay, it will just worsen the matter.

**Comment by SELFHELL** [comment]
Ill see what maints think, but I feel that making Royal Guards OVERPOWERING is a good thing and helps cement the authority of the monarch.

**Comment by RoboticVodka** [comment]
The skills seem a bit unnecessary as Royal Guard is pretty decent in that department already. To not make them too overpowering in the armor department my suggested changes to the proposed loadout would be suggesting the neck protection actually be an iron chain coif and their undershirt and pants also be iron chain. Makes it so if you really want to try to be invincible you'd need to upgrade your neck protection, under armor/pants, and get bracers while all the outwardly presenting stuff still carries the theme of the shining knight.

**Comment by BadAtThisGame302** [comment]
The Royal Guards do deserve something at least. They wear the same armor as the Garrison Guards and probably only get stat and skill boosts.

**Pull request closed without merging**